src/pages/MenuPage.js

```diff
@@ -38,96 +95,66 @@ export class MenuPage extends React.Component {
     database.removeListener(this.databaseListenerId);
   }
 
+  /**
+   * Method will try to match stringified customData object for current store
+   * against keys in MODULES, will return values of keys in MODULES
+   * that match: customData[keyInModules]data === 'true'
+   * @param {string}  customDataAsString  Store custom data as string
+   * @return {array}  result              The maxOrdersPerPeriod of the orderType
+   */
+  getUsedModules = customDataAsString => {
+    if (!customDataAsString || customDataAsString.length === 0) return [];
+
+    const customData = JSON.parse(customDataAsString);
+    const result = [];
+    const checkAndAddToResult = ([moduleName, moduleValues]) => {
+      if (!customData[moduleName]) return;
+      if (!customData[moduleName].data) return;
+      if (customData[moduleName].data === 'true') result.push(moduleValues);
+    };
+
+    Object.entries(MODULES).forEach(checkAndAddToResult);
+
+    return result.length > 0 && result;
```

Comment:
Inconsistent returns. If `result = []`, we return `false` and then the caller above calls `false.length`. Still undefined and falsey so fine but unintended?

Reply:
fixed: https://github.com/openmsupply/mobile/pull/868/commits/cdc1d9034ff584383b6407ad421f324e2a2bf9f2